saltychat/SaltyClient/VoiceManager.cs

```diff
@@ -346,6 +346,11 @@ private void OnMessage(dynamic message, dynamic cb)
             {
                 VoiceManager.IsTalking = pluginState.IsTalking;
 
+                if (VoiceManager.IsTalking)
+                    await Game.PlayerPed.Tasks.PlayFacialAnimation("face_human@gen_male@base", "mood_talking_normal");
+                else
+                    await Game.PlayerPed.Tasks.PlayFacialAnimation("face_human@gen_male@base", "mood_normal");
```

Comment:
If you `await` these, you will delay the event- and NUI callback calls.
You could discard the `Task`, because there is no use for it here anyways.

e.g.
```csharp
_ = Game.PlayerPed.Tasks.PlayFacialAnimation("face_human@gen_male@base", "mood_talking_normal");
```